snooty/rstspec.toml

```diff
@@ -580,6 +580,32 @@ help = """A list of languages that this guide supports."""
 # """
 content_type = "list"
 
+[directive.column]
+help = """column content for the n-column component"""
+content_type = "block"
+options.title = "string"
+
+[directive."multi-column"]
+help = """A two-column component for the homepage"""
```

Comment:
change to multi column component

Reply:
also, would add a note (or option) to note how each column would be presented in small vs big screen. the design shows each column being its own block